### ISSUE TITLE
:Unite taskwarrior/annotations improvement

### ISSUE THREAD

**Issue by linuxcaffe** [blakesweeney/unite-taskwarrior#35]
The /annotations list technically works, in that it generates a list of annotation content. Unfortunately, that listing is fairly disjointed, (except for a uuid) unreferenced and (imho) somewhat useless! 

To make this function worth having, I think a two-pane treatment (not unlike /notes with -auto-preview) would work well, putting a list of all tasks with annotations (one line per) on the left, and an -auto-preview of the selected tasks annotation(s) content on the right. The annotation preview should show timestamps too.

**Comment by blakesweeney** [comment]
Yeah this view isn't good. I'll start by improving the listing. Hopefully it's easier to make the the associated task clear. The two pane configuration will probably be the suggested configuration for it. I may be able to set it as default as well.

**Comment by blakesweeney** [comment]
I've reworked annotation formatting to put annotation on top and then task indented below it. The task is formatted using the standard formatter, but we strip off the initial `-` or `+`. I think this fixes it up quite a bit.

**Comment by linuxcaffe** [comment]
This IS an improvement, yet somehow, not the sort of thing one would immediately go to..

either;
a) sort by description.. idonno..

This will become more of a "thing" when multi-line annotations comes on line, and then, imho, the only effective annotations list, would look a lot like /notes with preview; a tasklist on the left (only tasksk with annotations) and previews on the right.. 

I'm sure there are other possibly advantageous layouts, just can't think of any :)
